Server/App/init_procedures.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import subprocess
+
+import MySQLdb
+
+from config import DBConfig
+
+"""
+Simple script that initializes the DB with stored procedures.
+"""
+
+
+db = MySQLdb.connect(host=DBConfig.HOST, user=DBConfig.USERNAME, passwd=DBConfig.PASSWORD)
+
+if not db:
+    logging.error('Could not connect to DB')
+    exit(1)
+
+try:
+    # Delete stored procedures in the Cue database.
+    cursor = db.cursor()
+    cursor.execute('DELETE FROM mysql.proc WHERE db LIKE \'{}\''.format(DBConfig.DB_NAME))
```

Comment:
Do we want to add type=procedure, or just delete stored functions in this step as well